src/roll20.pyj

```diff
@@ -362,6 +363,12 @@ def rollAttack(request, custom_roll_dice=""):
         damage_types = list(request["damage-types"])
         crit_damages = list(request["critical-damages"])
         crit_damage_types = list(request["critical-damage-types"])
+
+        # Ranger Ability Support
+        for dmgIndex, dmgType in enumerate(damage_types):
+            if damage_types[dmgIndex] in ["Colossus Slayer", "Planar Warrior"]:
+                damages[dmgIndex] = ROLL20_ADDL_DMG_QUERY.replace(/%1/, damage_types[dmgIndex]).replace(/%2/, damages[dmgIndex])

```

Comment:
Not a fan, but I think it's a smart way of doing it as a generic query. But I'd prefer `%dmg%` and `%dmgtype%` to just `%1` and `%2`
Also, I don't want to have a query on every roll for planar warrior, so it needs to just be a character sheet setting.